cmake/tests.cmake

```diff
@@ -12,4 +12,5 @@ function(register_test)
     COMMAND
       "$<TARGET_FILE:${TEST_TARGET}>"
       "--gtest_output=xml:${TESTRESULTS_DIRECTORY}/${TEST_NAME}_sponge_log.xml")
+  set_tests_properties(${TEST_NAME} PROPERTIES TIMEOUT 10)
```

Comment:
This is now the timeout for all our tests? Might be okay for now, but we at some point we need to allow to set this per test individually.

Reply:
I would also feel more comfortable having this line after the register_test call in OrbitSsh/CMakeLists.txt

Reply:
I looked briefly into making the timeout an default argument for the `register_test` function, but I couldn't make it work since I don't know enough about Cmake, so I am going to leave it for now

Reply:
done